src/backend/app_LOCAL_12238.js

```diff
@@ -0,0 +1,87 @@
+
+const express = require('express');
```

Comment:
Why do we have 3 version of this file?

Reply:
Oh, yeah
I believe I had a merge conflict in an earlier branch which I thought I had resolved.
I must have added the other files by mistake. 
Let me remove them